File: app_flutter/test/core/app_config_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:app_flutter/core/app_config.dart';

void main() {
  test('AppConfig.appDisplayName is non-empty', () {
    expect(AppConfig.appDisplayName, isNotEmpty);
  });

  test('AppConfig.appDisplayName equals AppConfig.title', () {
    expect(AppConfig.appDisplayName, equals(AppConfig.title));
  });
}

File: web_react/src/config/app_config.test.ts
import { APP_DISPLAY_NAME } from './app_config';

describe('AppConfig', () => {
  it('APP_DISPLAY_NAME is a non-empty string', () => {
    expect(APP_DISPLAY_NAME).toBeTruthy();
    expect(typeof APP_DISPLAY_NAME).toBe('string');
    expect(APP_DISPLAY_NAME.length).toBeGreaterThan(0);
  });
});
